script/DeploySepolia.s.sol

```diff
@@ -90,7 +90,7 @@ contract DeploySepoliaScript is Script, Deployers, MockStakingV1Deployer {
                 votingThresholdFactor: VOTING_THRESHOLD_FACTOR,
                 minClaim: MIN_CLAIM,
                 minAccrual: MIN_ACCRUAL,
-                epochStart: uint32(block.timestamp - VESTING_EPOCH_START),
```

Comment:
This makes `VESTING_EPOCH_START` unused. Also, `VESTING_EPOCH_DURATION` was never used in the first place.